rollup.config.js

```diff
@@ -29,7 +29,7 @@ export default {
 			// we'll extract any component CSS out into
 			// a separate file — better for performance
 			css: css => {
-				css.write(`public/bundle.${cacheBust}.css`)
+				css.write(`bundle.${cacheBust}.css`)
```

Comment:
It seems odd to me that this CSS bundle was moved, but there's no corresponding change to something like an index.html to load it from the new location. Is that autogenerated during build time somewhere?

Reply:
It isn't moving the bundle. The rollup changed it to use the output directory as the default path, so I needed to remove the `public/` to keep the file in the same place.